### ISSUE TITLE
Bug with Image Normalization?

### ISSUE THREAD

**Issue by Nik-V9** [jasonyzhang/RayDiffusion#24]
Hey @jasonyzhang, I was going through the code and I see that all the dataloaders use the following image normalization: https://github.com/jasonyzhang/RayDiffusion/blob/f53303a2983f04b1ce6e60c967cecd1e1695b74f/ray_diffusion/dataset/co3d_v2.py#L309

However, the recommend normalization for DINOv2 is ImageNet normalization: https://github.com/facebookresearch/dinov2/blob/e1277af2ba9496fbadf7aec6eba56e8d882d1e35/dinov2/data/transforms.py#L42

I wanted to check if this is intentional?

**Comment by jasonyzhang** [comment]
Hi Nikhil,

That is a bug. We should've used ImageNet normalization. Our pre-trained models are trained with this incorrect normalization. If re-training, the performance will probably be better with the imagenet normalization.

**Comment by Nik-V9** [comment]
Yup, I expect the performance to be considerably better with the correct normalization!

The performance is already great anyways ;)
